pages/desktop/consumer_pages/base.py

```diff
@@ -68,9 +71,15 @@ def set_region(self, region):
         self.wait_notification_box_not_visible()
 
     def login(self, user=None):
-        from pages.fxa import FirefoxAccounts
-        fxa = FirefoxAccounts(self.testsetup)
-        fxa.login_user(user)
+        credentials = isinstance(user, MockUser) and user or \
+                    self.testsetup.credentials.get(user, FxATestAccount(use_prod=False).create_account())
+        from fxapom.pages.desktop.sign_in import SignIn
+        fxa_login = SignIn(self.testsetup)
+        if user=='default':
+            fxa_login.sign_in(credentials['email'], credentials['password'])
+        elif user=='new':
+            fxa_login.sign_in(credentials.email, credentials.password)
+
```

Comment:
If that's the case -- note I don't have domain knowledge for this project -- I would suggest changing `user=None` to `user=default` and then adding an `else` conditional that returns an exception.